tests/layers/vllm/test_compressed_tensors_moe.py

```diff
@@ -80,7 +80,8 @@ def _ref_math_in_bf16(w1, w2, w3, x, router_logits, top_k):
     return out
 
 
-def test_fused_moe_method():
+@pytest.mark.parametrize("use_ep", [False, True])
```

Comment:
can you add more parameters to cover more cases. you can refer to other unit tests: https://github.com/vllm-project/tpu-inference/blob/2ffab1f900bc30cc675a9dab57f7fa97c944eea3/tests/layers/vllm/test_mxfp4.py#L107-L115